src/task1.java

```diff
@@ -0,0 +1,26 @@
+import java.util.Scanner;
+import java.util.Arrays;
+
+public class task1 {
+    public static void main(String[] args) {
+        isArraySortedAscending();
+    }
+    public static int[] array() {
+        int n;
+        Scanner sc = new Scanner(System.in);
+        System.out.print("How many numbers should array contain: ");
+        n = sc.nextInt();
+        System.out.print("Enter your numbers: ");
+        int[] array = new int[n];
+        for (int i = 0; i < n; i++) {
+            array[i] = sc.nextInt();
+        }
+        return array;
+    }
+    public static void isArraySortedAscending() {
+        int[] unsortedArray = array();
+        int[] sortedArray = Arrays.copyOf(unsortedArray, unsortedArray.length);
```

Comment:
you're copying the array and then sorting it to check if the original array is sorted, this is unnecessary. Instead, you can compare adjacent elements in the original array
**isArraySortedAscending()** should check if each element is smaller than or equal to the next, which determines if the array is sorted in ascending order without the need to copy and sort the array